ui/Dockerfile.dev

```diff
@@ -1,14 +1,14 @@
 # Note: This is the dockerfile for development purposes
 
 # We are aware that some dev dependencies are broken in node 14, for example chokidar
-# npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. 
+# npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+.
 FROM node:14
 
-# Install python 2, needed for npm packages
-RUN apt-get update && apt-get -y install python
+# Install python 3, needed for npm packages
+RUN apt-get update && apt-get -y install python3
```

Comment:
I would hope that `apt` installs Python 3 when you say `install python`, the special `python3` alias seems a few years out of date at this point. (It was created so people could keep Python 2 installed as `python`.)